language/tools/move-package/src/resolution/lock_file/schema.rs

```diff
@@ -20,24 +20,44 @@ use toml::value::Value;
 pub const VERSION: u64 = 0;
 
 #[derive(Deserialize)]
-pub struct Dependencies {
-    #[serde(rename = "dependency")]
-    dependencies: Option<Vec<Dependency>>,
+pub struct Packages {
+    #[serde(rename = "package")]
```

Comment:
It's named for the singular because it's intended to be used with the `[[...]]` TOML syntax:

```
[[move.package]]
name = "..."
```

This is similar to how it looks in Cargo.lock as well.